decorator.js

```diff
@@ -172,10 +184,20 @@ class Decorator {
     return list;
   }
 
+  generateComment(name) {
+    if (this.isCommentEnabled) {
+      const id = hash(name);
```

Comment:
hash should include a corresponding suite name or filename because there might be tests with the same name